DEVELOPMENT.md

```diff
@@ -208,7 +206,7 @@ kubectl patch configmap/config-network \
 ```
 
 If you want to choose another Ingress solution, you can follow step 3 in the
-[Knative installation doc](https://knative.dev/docs/install/any-kubernetes-cluster/#installing-the-serving-component)
+[Knative installation doc](https://knative.dev/docs/admin/install/serving/install-serving-with-yaml/)
```

Comment:
while we're fixing this, we could actually link right to the heading here: https://knative.dev/docs/admin/install/serving/install-serving-with-yaml/#install-a-networking-layer

Reply:
fixed, thanks for the putting the better link